org.eclipse.scout.rt.rest.jersey.test/src/test/java/org/eclipse/scout/rt/rest/jersey/client/AbstractRestClientHelperTest.java

```diff
@@ -60,4 +68,38 @@ public void testLazyClientInstance() {
     Client client2 = BEANS.get(RestClientProxyFactory.class).unwrap(restClientHelper.client());
     assertEquals("expect same client instance on multiple calls", client1, client2);
   }
+
+  @Test
+  public void testDuplicatedContextResolverAsBean() {
+    IBean<?> bean = BeanTestingHelper.get().registerBean(new BeanMetaData(ObjectMapperResolver2.class));
```

Comment:
Registering/unregistering bean shouldn't be required if JerseyTestRestClientHelperEx overrides getContextResolversToRegister anyway